docs/resources/cpan.md.erb

```diff
@@ -0,0 +1,82 @@
+---
+title: About the cpan Resource
+---
+
+# cpan
+
+Use the `cpan` InSpec audit resource to test Perl modules that are installed by system packages or the CPAN installer.
+
+## Syntax
+
+A `cpan` resource block declares a package and (optionally) a package version:
+
+    describe cpan('package_name') do
+      it { should be_installed }
+    end
+
+where
+
+* `'package_name'` is the name of the package, such as `'DBD::Pg'`
+* `be_installed` tests to see if the package described above is installed
+
+
+## Matchers
+
+This InSpec audit resource has the following matchers:
+
+### be
+
+<%= partial "/shared/matcher_be" %>
+
+### be_installed
+
+The `be_installed` matcher tests if the named package is installed on the system:
+
+    it { should be_installed }
+
+### cmp
```

Comment:
You can remove the boilerplate matchers - we have since removed them from all other resource documentation since you submitted this.  Lines 27-30 and lines 37-52 can be deleted!